### PR TITLE
HTS-133: Added scalastyle-config and scalastyle and scoverage sbt plu…

### PR DESCRIPTION
**Pull request by Jackie1988**
…gins

### CODE REVIEW

File: project/MicroService.scala [hunk 1]
```diff
@@ -22,6 +22,18 @@ trait MicroService {
   lazy val plugins: Seq[Plugins] = Seq.empty
   lazy val playSettings: Seq[Setting[_]] = Seq.empty
 
+  lazy val scoverageSettings = {
+    import scoverage.ScoverageKeys
+    Seq(
+      // Semicolon-separated list of regexs matching classes to exclude
+      //ScoverageKeys.coverageExcludedPackages := "<empty>;Reverse.*;.*AuthService.*;models/.data/..*;.*BuildInfo.*;prod.Routes;app.Routes;testOnlyDoNotUseInAppConf.Routes;iht.controllers.wraith.*;iht.controllers.testonly.*;iht.views.html.testOnly.*;wraith.Routes;taxreturn.Routes;registration.Routes;iht.auth.*;iht.controllers.auth.*;iht.connector.*;iht.config.*;iht.metrics.*;iht.views.html.ihtHelpers.standard.*;iht.models.Person;iht.models.TaxIdsFormat;iht.forms.testonly.*;",
+      ScoverageKeys.coverageMinimum := 80,
+      ScoverageKeys.coverageFailOnMinimum := true,
```
Comment (sureshhmrc) [inline project/MicroService.scala]: we should set this to false initially and until we reach 80

File: project/plugins.sbt [hunk 2]
```diff
@@ -2,10 +2,16 @@ resolvers += Resolver.url("hmrc-sbt-plugin-releases", url("https://dl.bintray.co
...
+
 addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "1.6.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "0.9.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "1.0.0")
 
 addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.5.12")
+
+addSbtPlugin("org.scoverage" %% "sbt-scoverage" % "1.3.3")
```
Comment (sureshhmrc) [inline project/plugins.sbt]: the recent version is 1.5.0 ?

File: project/MicroService.scala (hunk 1 above)
Comment (Jackie1988) [inline project/MicroService.scala]: ok I am going to find out what our coverage is currently

File: project/plugins.sbt (hunk 2 above)
Comment (Jackie1988) [inline project/plugins.sbt]: Yes you're right, I will update

**Pull request merged**